composer.json

```diff
@@ -4,7 +4,7 @@
 	"license": "GPL-2.0-or-later",
 	"config": {
 		"platform": {
-			"php": "7.2"
+			"php": "7.2.5"
```

Comment:
```suggestion
			"php": "7.2.0"
```

Reply:
Hey, thanks for the feedback :) I used the commit https://github.com/TYPO3/TYPO3.CMS/commit/33471eff99a5b5ccf79e74b6ce434474bb20b669 and setting https://github.com/TYPO3/typo3/blob/10.4/composer.json#L29 as a base, which recommends 7.2.5 in order to catch wanted Symfony dependencies. Dependency `^7.2` would match the platform setting `7.2.5`. 

I would prefer to streamline the setting in both packages. Otherweise the tarball release has other dependencies included, than then composer-based basedistribution. This looks a bit flaky.

However, since Benni Mack said in the commit that the fetched Symfony dependencies have features we don't actually use, I'm okay with setting `7.2.0` here in order to match the mentioned system requirements in https://get.typo3.org/version/10#system-requirements. I can change the PR asap.

What are your thoughts/decision on this? 😀